server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/CBApplication.java

```diff
@@ -201,13 +203,20 @@ protected void startServer() {
             if (!loadServerConfiguration()) {
                 return;
             }
+
             if (CommonUtils.isEmpty(this.getAppConfiguration().getDefaultUserTeam())) {
                 throw new DBException("Default user team must be specified");
             }
         } catch (DBException e) {
             log.error(e);
             return;
         }
+        // Set default preferences
+        PrefUtils.setDefaultPreferenceValue(DBWorkbench.getPlatform().getPreferenceStore(),
+            ModelPreferences.UI_DRIVERS_HOME,
+            getServerConfiguration().getDriversLocation());
+        CBPlatform.getInstance().refreshApplicableDrivers();
```

Comment:
We also call refreshApplicableDrivers in CBPlatform.initialize. Maybe keep in one place only?